cdm-test/src/test/java/ucar/nc2/util/net/TestHTTPSession.java

```diff
@@ -89,27 +86,27 @@ public TestHTTPSession()
         System.out.println("Test: HTTPSession.setGlobalUserAgent(" + GLOBALAGENT + ")");
 
         HTTPSession.debugHeaders(false);
-
         HTTPSession.setGlobalUserAgent(GLOBALAGENT);
         try (HTTPSession session = HTTPFactory.newSession(TESTURL1)) {
+            List<Header> agents = null;
             HTTPMethod method = HTTPFactory.Get(session, TESTURL1);
             method.execute();
             // Use special interface to access the request
             // Look for the user agent header
-            List<Header> agents = HTTPSession.debugRequestInterceptor().getHeaders(HTTPSession.HEADER_USERAGENT);
+            agents = HTTPSession.debugRequestInterceptor().getHeaders(HTTPSession.HEADER_USERAGENT);
             Assert.assertFalse("User-Agent Header not found", agents.size() == 0);
             Assert.assertFalse("Multiple User-Agent Headers", agents.size() > 1);
             Assert.assertTrue(String.format("User-Agent mismatch: expected %s found:%s",
                     GLOBALAGENT, agents.get(0).getValue()),
                     GLOBALAGENT.equals(agents.get(0).getValue()));
             System.out.println("*** Pass: set global agent");
+            //method.close();
 
```

Comment:
Is there a particular reason to check in with this commented out (and not just removed)?